scripts/cm-rgb-monitor

```diff
@@ -45,18 +20,21 @@ def print_available_sources(ctx,value):
 @click.option("--bg-color", "bgColor", default="#00FFFF", help="Background LED's color")
 @click.option("--cpu-color", "cpuColor", default="#FFA500", help="Color of the cpu load LED's")
 @click.option("--brightness", type=click.IntRange(1, 5, clamp=True), default=3)
-@click.option("--interval", type=click.FloatRange(0.01, 60, clamp=True), default=0.2)
+@click.option("--interval", type=click.FloatRange(0.01, 60, clamp=True), default=0.1)
```

Comment:
Is this default value change intentional ?

Reply:
No, default values was only my personal choice and found its way into my PR as I didn't change it back.

Reply:
Ok, I'll merge it, but I'll probably change the default back as I found that at 0.1s ring can flicker noticeably in my opinion.

Thanks for this awesome feature :+1: